useIsShake.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Added file to instantiate custom hook for logic calculate a shake and set state variables in response to the registration of a shake for use throughout app.